community-membership.md

```diff
@@ -113,26 +113,26 @@ only be doing issues triage and have no approval rights.
 ### Requirements
 
 The following apply to the part of the codebase for which one would be an
-approver in the `CODEOWNER` files.
+approver in the `CODEOWNERS` files.
 
-- Reviewer of the codebase for at least 3 months
-- Reviewer for at least 10 substantial PRs to the codebase
-- Reviewed or merged at least 30 PRs to the codebase
+- Reviewer of the codebase for at least 1 month
+- Reviewer for or contributor of at least 10 substantial PRs to the codebase
```

Comment:
nit: in the next line we don't have the "for" or "of" prepositions, could remove here as well.

Reply:
non-nit: how do we define "substantial"?

Reply:
We don't know how to do that, I'd suggest deferring that to maintainer discretion, but my assumption is non-trivial e.g. not one-line commits, but any kinds of more major refactorings or feature additions...